client/src/main/java/org/spine3/base/ListStringifier.java

```diff
@@ -0,0 +1,172 @@
+    private final Splitter splitter;
+
+    /**
+     * Creates a {@code ListStringifier}.
+     *
+     * <p>The {@code DEFAULT_ELEMENT_DELIMITER} is used for element
+     * separation in {@code String} representation of the {@code List}.
+     *
+     * @param listGenericClass the class of the list elements
+     */
+    ListStringifier(Class<T> listGenericClass) {
+        super();
+        this.listGenericClass = listGenericClass;
+        this.delimiter = DEFAULT_ELEMENT_DELIMITER;
+        escaper = Stringifiers.createEscaper(delimiter);
+        splitter = Splitter.onPattern(createElementDelimiterPattern(delimiter));
+    }
+
+    /**
+     * Creates a {@code ListStringifier}.
+     *
+     * <p>The specified delimiter is used for element separation
+     * in {@code String} representation of the {@code List}.
+     *
+     * @param listGenericClass the class of the list elements
+     * @param delimiter        the delimiter for the passed elements via string
+     */
+    ListStringifier(Class<T> listGenericClass, char delimiter) {
+        super();
+        this.listGenericClass = listGenericClass;
+        this.delimiter = delimiter;
+        escaper = Stringifiers.createEscaper(delimiter);
+        splitter = Splitter.onPattern(createElementDelimiterPattern(delimiter));
+    }
+
+    private static String createElementDelimiterPattern(char delimiter) {
+        return Pattern.compile("(?<!\\\\)\\\\\\" + delimiter)
+                      .pattern();
+    }
+
+    @Override
+    protected String toString(List<T> list) {
+        final Function<T, String> convertFunction = new Function<T, String>() {
+            @Nullable
+            @Override
+            public String apply(@Nullable T input) {
+                checkNotNull(input);
+                return Stringifiers.toString(input, listGenericClass);
+            }
+        };
+        final List<String> convertedList = Lists.transform(list, convertFunction);
+        final Function<String, String> function = new Function<String, String>() {
+            @Nullable
+            @Override
+            public String apply(@Nullable String input) {
+                checkNotNull(input);
+                return QuotedItem.quote(input);
+            }
+        };
+        final List<String> quotedItems = Lists.transform(convertedList, function);
+        final String result = Joiner.on(delimiter)
+                                    .join(quotedItems);
+        return result;
+    }
+
+    @Override
+    protected List<T> fromString(String s) {
+        final String escapedString = escaper.escape(s);
+        final List<String> elements = newArrayList(splitter.split(escapedString));
+        final Function<String, String> unquoteFunction = new Function<String, String>() {
+            @Nullable
+            @Override
+            public String apply(@Nullable String input) {
+                checkNotNull(input);
+                return QuotedItem.unquote(input);
```

Comment:
@IlliaShepilov Please see [that](https://google.github.io/guava/releases/17.0/api/docs/com/google/common/base/Converter.html#andThen(com.google.common.base.Converter)) on chaining.